File: gs/gs4719/practice/exam/p/p03.py
def p03(x=123):
    reverse=None
    # ↓程式區域↓
    # 將數字轉換為字串，然後反轉字串，最後再轉回整數
    if x >= 0:
        reverse = int(str(x)[::-1])
    else:
        reverse = -int(str(-x)[::-1])

    # ↑程式區域↑
    return reverse

# 測試程式碼
if __name__ == "__main__":
    result = p03(123)
    print(result)

File: ce/CE1004/P03/P03.java
package CE1004.P03;
class Student {
  private String name;
  private int score;

  public Student(String name, int score) {
    this.name = name;
    this.score = score;
  }
  public String getName() {
    return name;
  }
  public int getScore() {
    return score;
  }
  public void changeScore(int newScore) {
    this.score = newScore;
  }
}

public class P03 {
  public static void main(String[] args) {
    // 以下不要改
    Student putin = new Student("Putin", 59);
    System.out.println(putin.getName() + " before: " + putin.getScore());
    putin.changeScore(40);
    System.out.println(putin.getName() + " after: " + putin.getScore());
//        putin.score = 100;
  }
}